pyuavcan/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -16,42 +24,174 @@
 class PythonCANMedia(_media.Media):
     """
     A media interface adapter for `python-can <https://github.com/hardbyte/python-can>`_.
-    This is a stub, the implementation is missing. Please submit patches!
+      * Usage example for PCAN-USB channel 1 (bitrate = 500k, mtu = 8, Node-ID = 10): 
+          --tr="CAN(can.media.pythoncan.PythonCANMedia('pcan','PCAN_USBBUS1',5000000,8),10)"
+      * Usage example for Kvaser channel 0 (bitrate = 500k, mtu = 8, Node-ID = 10): 
+          --tr="CAN(can.media.pythoncan.PythonCANMedia('kvaser','0',5000000,8),10)"
```

Comment:
That's not quite how you write reStructuredText. This is how you do it so that it's rendered correctly:

```suggestion

    - Usage example for PCAN-USB channel 1 (bitrate = 500k, mtu = 8, Node-ID = 10)::

        --tr="CAN(can.media.pythoncan.PythonCANMedia('pcan','PCAN_USBBUS1',5000000,8),10)"

    - Usage example for Kvaser channel 0 (bitrate = 500k, mtu = 8, Node-ID = 10)::

        --tr="CAN(can.media.pythoncan.PythonCANMedia('kvaser','0',5000000,8),10)"
```